pkg/webhook/vlanconfig/mutator.go

```diff
@@ -0,0 +1,125 @@
+package vlanconfig
+
+import (
+	"encoding/json"
+	"fmt"
+	ctlcorev1 "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	"github.com/yaocw2020/webhook/pkg/types"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"reflect"
+
+	networkv1 "github.com/harvester/harvester-network-controller/pkg/apis/network.harvesterhci.io/v1beta1"
+	"github.com/harvester/harvester-network-controller/pkg/utils"
+)
+
+type vlanConfigMutator struct {
+	types.DefaultMutator
+
+	nodeCache ctlcorev1.NodeCache
+}
+
+var _ types.Mutator = &vlanConfigMutator{}
```

Comment:
Question: what's the usage of this unamed global variable?

Reply:
Indicate that `&vlanConfigMutator{}` must implement the interface `types.Mutator`.